### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by cookieman12345** [MeteorDevelopment/meteor-client#2379]
### Describe the bug

meteor client v0.5.0 build #1496 isn't working, it gives me this error code.

The game crashed whilst initializing game
Error: java.lang.NullPointerException: Cannot invoke "meteordevelopment.meteorclient.renderer.text.TextRenderer.begin(double, boolean, boolean)" because the return value of "meteordevelopment.meteorclient.renderer.text.TextRenderer.get()" is null
Exit Code: -1

### Steps to reproduce

```bash
loading up fabric loader 1.19 with the mod meteor client v0.5.0 build #1496
```

### Meteor Version

meteor client v0.5.0 build #1496

### Minecraft Version

1.19

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by TheKidThatCodes** [comment]
Duplicate of #2376?

**Comment by TheKidThatCodes** [comment]
Possible Duplicate of #2377

**Comment by arlomcwalter** [comment]
can you send the whole log rather than crash report (.minecraft/logs)

**Comment by DarkReaper231** [comment]
I got same error on Linux, it is trying to use fonts from non existing directory. On Windows it's `C:\Users\USER\AppData\Local\Microsoft\Windows\Fonts` On Linux `it's /home/USER/.local/share/fonts`

It's trying to fetch JetBrains fonts from these folders.

**Comment by DarkReaper231** [comment]
> [latest.log](https://github.com/MeteorDevelopment/meteor-client/files/8894567/latest.log)

Temporary fix is to download JetBrains font from here https://www.jetbrains.com/lp/mono/ and put all the ttf files into the directory mentioned above

**Comment by DarkReaper231** [comment]
> [latest.log](https://github.com/MeteorDevelopment/meteor-client/files/8894567/latest.log)

Btw I suggest you to rename the issue so it contains some information like
[Bug] Meteor is trying to use JetBrains fonts from the OS even when they are not installed

**Comment by DarkReaper231** [comment]
Seems fixed in #76b3

**Comment by angrydestreyer9** [comment]
> ill test right now... i'm assuming by commit you mean the latest github release?

how would I test it? where is hte download... new to github

**Comment by Wide-Cat** [comment]
fixed in 76b332689d739fc1e44989ca5f1dd1cd09c34332
